src/chunk_adaptive.c

```diff
@@ -146,7 +146,7 @@ minmax_heapscan(Relation rel, Oid atttype, AttrNumber attnum, Datum minmax[2])
 	if (NULL == tce || !OidIsValid(tce->cmp_proc))
 		elog(ERROR, "no comparison function for type %u", atttype);
 
-	scan = heap_beginscan(rel, GetTransactionSnapshot(), 0, NULL);
+	scan = table_beginscan(rel, GetTransactionSnapshot(), 0, NULL);
```

Comment:
Same as above: maybe this should be called `relation_beginscan`?

Reply:
also a Postgres name